artiq/frontend/artiq_master.py

```diff
@@ -26,7 +26,7 @@
 
 
 def get_argparser():
-    parser = argparse.ArgumentParser(description="ARTIQ master")
+    parser = argparse.ArgumentParser(description="ARTIQ master. Requires access to device database.")
```

Comment:
Not sure if this is the place to mention that

Reply:
My thought here is it would be nice to have an upfront straightforward clarification somewhere on which front-end commands need to be run with the device database and which need to be run with the master, since I remember it's something that caused me some real confusion at first. If we don't want it in the argparse then I suggest I'll `:nodescription:` to delink the descriptions in the reference page and connect it there... open to either option. 

But anecdotally e.g. I remember getting confused between the `-D IP` of `artiq_coremgmt` and the `-s IP` of `artiq_client`, although obviously with more context these are not remotely the same thing, and until recently I was under the mistaken impression `artiq_ctlmgr` needed to be run with a `device_db.py`; it seems more accessible to separate "needs contact with core device" and "needs contact with `artiq_master`" somewhere really nice and obvious. (It didn't help that 'the master' is also sometimes the core device, if you've been reading about DRTIO... but the manual makes itself pretty clear in. re. that distinction now, I think.)